### DISCUSSION TITLE
Feature request : import to a selected tag

### DISCUSSION THREAD

**Discussion by antoineschmitt** [standardnotes/forum#2507] [Suggestions] [answered]
I am trying to import my finely organized notes database from the "Notebooks" app, and all of my notes get into the "All" tag, where I had them (more than 200 notes) well organized into folders.
I would like to be able to export my folders one by one and attach each folder to a Tag.

For this, one simple solution would be to be able to "import from backup" to the currently selected tag.

**Comment by atmoio** [comment]
Hmm..it's an interesting idea. A little too specific, but worth considering.

**Comment by zarazum** [comment]
It would be nice to choose what tags to apply to imported notes, even by entering comma-separated list.

**Comment by atmoio** [comment]
I think we would add this feature to the converter itself. Wouldn't be too hard, we already have the code for it, we just need an input in the UI: https://github.com/standardnotes/sntools/blob/master/lib/sntools.js#L77

**Comment by zarazum** [comment]
Well. What if I have a tag named WORDS and then while importing I'll enter WORDS into that input. 

Seems that converter will create second WORDS tag. Such a behaviour will not be the desired result.

**Comment by myreli** [comment, marked as answer]
The new importer tool auto-tags imported notes with the time of import, so I will mark it resolved. 

<img width="1257" alt="image" src="https://user-images.githubusercontent.com/17554234/218586584-ff92dd94-0e71-4103-914d-f60f351bfe2d.png">
